pkg/kubectl/cmd/top_pod.go

```diff
@@ -152,7 +157,7 @@ func (o TopPodOptions) RunTopPod() error {
 	if err != nil {
 		return err
 	}
-	return o.Printer.PrintPodMetrics(metrics, o.PrintContainers, o.AllNamespaces)
+	return o.Printer.PrintPodMetrics(metrics, o.PrintContainers, o.AllNamespaces, api.ResourceName(o.Sort))
```

Comment:
api.ResourceName is an object that is just a string. I figured it made more sense semantically to use that type instead of `string`